operator-common/src/main/java/io/strimzi/operator/common/operator/resource/ResourceDiff.java

```diff
@@ -30,15 +30,13 @@ public ResourceDiff(String resourceKind, String resourceName, T current, T desir
             String pathValue = d.get("path").asText();
 
             if (ignorableFields.matcher(pathValue).matches()) {
-                log.debug("Ignoring {} {} diff {}", resourceKind, resourceName, d);
+                LOGGER.debugCr(reconciliation, "Ignoring {} {} diff {}", resourceKind, resourceName, d);
                 continue;
             }
 
-            if (log.isDebugEnabled()) {
-                log.debug("{} {} differs: {}", resourceKind, resourceName, d);
-                log.debug("Current {} {} path {} has value {}", resourceKind, resourceName, pathValue, lookupPath(source, pathValue));
-                log.debug("Desired {} {} path {} has value {}", resourceKind, resourceName, pathValue, lookupPath(target, pathValue));
-            }
+            LOGGER.debugCr(reconciliation, "{} {} differs: {}", resourceKind, resourceName, d);
+            LOGGER.debugCr(reconciliation, "Current {} {} path {} has value {}", resourceKind, resourceName, pathValue, lookupPath(source, pathValue));
+            LOGGER.debugCr(reconciliation, "Desired {} {} path {} has value {}", resourceKind, resourceName, pathValue, lookupPath(target, pathValue));
```

Comment:
Should we keep the `if`? I assume we did this to avoid wasting resources. Or is that not possible anymore with the markets?